docs/manifest.json

```diff
@@ -187,6 +187,15 @@
           "title": "Upgrade Guide",
           "path": "/docs/upgrading.md"
         },
+        {
+          "title": "Migrating",
```

Comment:
What about `Migrating to Next.js`?

Reply:
I agree that `Migrating to Next.js` is better because migration could also mean "updating from older versions of Next.js".